framework/basehandlers_test.py

```diff
@@ -522,7 +519,6 @@ def test_get__html_page(self):
 
     self.assertIn('Hi testing', actual_html)
     self.assertEqual(200, actual_status)
-    self.assertIn('Access-Control-Allow-Origin', actual_headers)
```

Comment:
It would be better to use assertNotIn().  We don't want to "not care" if the header is there, we want to affirm that it is absent.

Reply:
Yea that's right. Thanks!